src/main/java/com/crafter/springboottechiteasy/mappers/TelevisionMapper.java

```diff
@@ -0,0 +1,87 @@
+package com.crafter.springboottechiteasy.mappers;
+
+import com.crafter.springboottechiteasy.Dtos.TelevisionInputDto;
+import com.crafter.springboottechiteasy.Dtos.TelevisionOutputDto;
+import com.crafter.springboottechiteasy.Dtos.TelevisionSalesDto;
+import com.crafter.springboottechiteasy.models.Television;
+
+public class TelevisionMapper {
+    public static TelevisionOutputDto TelevisionToOutput(Television television) {
+        TelevisionOutputDto dto = new TelevisionOutputDto();
+        dto.setId(television.getId());
+        dto.setType(television.getType());
+        dto.setBrand(television.getBrand());
+        dto.setName(television.getName());
+        dto.setPrice(television.getPrice());
+        dto.setAvailableSize(television.getAvailableSize());
+        dto.setRefreshRate(television.getRefreshRate());
+        dto.setScreenType(television.getScreenType());
+        dto.setScreenQuality(television.getScreenQuality());
+        dto.setSmartTv(television.getSmartTv());
+        dto.setWifi(television.getWifi());
+        dto.setVoiceControl(television.getVoiceControl());
+        dto.setHdr(television.getHdr());
+        dto.setBluetooth(television.getBluetooth());
+        dto.setAmbilight(television.getAmbilight());
+        dto.setOriginalStock(television.getOriginalStock());
+        dto.setSold(television.getSold());
+        return dto;
+    }
+
+    // is output naar television nog nodig, nu er een speciale input DTO is voor PUT en POST?
+    public static Television OutputToTelevision(TelevisionOutputDto dto) {
```

Comment:
Deze is niet nodig, Ik verwacht dat je nooit het probleem gaat hebben dat je een outputdto naar model moet mappen :)